RELEASE_NOTES.md

```diff
@@ -39,6 +39,11 @@
 
 - Drop `Averager` from `FormulaEngine`.
 
+- The PowerDistributingActor now supports n:m relations between inverters and
+  batteries.
+  This means that one or more inverters can be connected to one or more batteries.
```

Comment:
I would also mention this affects the battery pool, as most users don't user the power distributing actor directly, and not use 80-chars limit, as it doesn't get properly rendered in GitHub release notes otherwise.

```suggestion
- The `PowerDistributingActor` (and thus the battery pools too) now supports *n:m* relations between inverters and batteries.

  This means that one or more inverters can be connected to one or more batteries.
```

Reply:
It is just the "control" methods in the battery pool.  Not all of it yet.